src/components/ec-currency-input/ec-currency-input.vue

```diff
@@ -148,6 +150,9 @@ export default {
       type: Boolean,
       default: false,
     },
+    currenciesTooltipInfo: {
```

Comment:
We should avoid meaningless words in object names like data, detail, info, etc. These are code smell when naming is hard. I see that you have also currenciesTooltip as a computed prop, that's why this one is an `...Info`. It's a prop, a public interface of the component so it should have the best comprehensive name for somebody who's going to use the component, so name this one `currenciesTooltip`.

The computed prop might have name like `currenciesTooltipOptions` or `currenciesTooltipConfig`, because it's just internal computed prop and it's easy to somebody to have look in the same file what's the meaning.

2nd problem with naming here:
This tooltip is only visible if currency is disabled. I think better name would be `currenciesDisabledReason` or `disabledCurrenciesTooltip`. When I saw `currenciesTooltip` for the first time, I thought it's displayed all the time.